### DISCUSSION TITLE
Is Xcode required?

### DISCUSSION THREAD

**Discussion by darsnack** [Homebrew/discussions#614] [Getting started] [answered]
As I understand it, Homebrew needs CLT. Does this mean I can uninstall Xcode and install just the CLT? Of course, certain casks might require Xcode itself, but if I don't have those, then I should be fine?

**Comment by carlocab** [comment, marked as answer]
Yes, that's about right. Though, I'm not aware of any casks that require Xcode. Some formulae do, but that's typically only required at build time.
